sklearn/utils/_param_validation.py

```diff
@@ -189,7 +196,8 @@ def wrapper(*args, **kwargs):
             )
 
             try:
-                return func(*args, **kwargs)
+                with config_context(skip_parameter_validation=skip_nested_validation):
```

Comment:
Same comment here regarding `prefer_skip_nested_validation` and considering the global config.